packages/alea-frontend/pages/api/access-control/add-remove-member.ts

```diff
@@ -8,15 +8,19 @@ import {
 } from '../comment-utils';
 import { recomputeMemberships } from './recompute-memberships';
 
-export default async function handler(req: NextApiRequest, res: NextApiResponse) {
-  if (!checkIfPostOrSetError(req, res)) return;
-  const memberId = req.body.memberId as string;
-  const aclId = req.body.aclId as string;
-  const isAclMember = req.body.isAclMember as boolean;
-  const toBeAdded = req.body.toBeAdded as boolean;
+export async function addRemoveMember(
```

Comment:
Fix this function. see
https://github.com/KWARC/ALeA/blob/main/backend_and_general_guidelines.md#27-returning-api-errors-outside-handler